packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -69,8 +69,12 @@ const findInProjectThenSDK = (pkg) => {
     // changes this file's location relative to the package root.
     const candidates = [
         resolve(projectDir, 'node_modules', pkg),
+        // Inside packages/pwa-kit-dev
         resolve(__dirname, '..', '..', 'node_modules', pkg),
-        resolve(__dirname, '..', '..', '..', 'node_modules', pkg)
+        resolve(__dirname, '..', '..', '..', 'node_modules', pkg),
+        // Inside packages/pwa-kit-react-sdk
+        resolve(__dirname, '..', '..', '..', 'pwa-kit-react-sdk', 'node_modules', pkg),
+        resolve(__dirname, '..', '..', '..', '..', 'pwa-kit-react-sdk', 'node_modules', pkg)
```

Comment:
To fix a ModuleNotFound error, because webpack was looking for `react-query` and couldn't find it. 
- In V2, the project `retail-react-app` does not use react-query.
- and `pwa-kit-dev` does not have it installed either → but `pwa-kit-react-sdk` does have it

Reply:
The function `findInProjectThenSDK()` looks for a dependency first in the project and then in the SDK. Historically, there was only 1 SDK package. But starting with V2, it was split into multiple packages. Thus, I think it's all right to also search within pwa-kit-react-sdk in this case.

Reply:
This change should only impact development within the monorepo, so it's potentially just a footgun internally. Given the relatively minimal development on v2, I think that this is an okay trade off, to unblock release now. ✅